app/src/main/java/sample/okta/android/browser/BrowserViewModel.kt

```diff
@@ -36,7 +36,7 @@ class BrowserViewModel : ViewModel() {
         viewModelScope.launch {
             _state.value = BrowserState.Loading
 
-            val webAuthenticationClient = CredentialBootstrap.oidcClient.createWebAuthenticationClient()
+            val webAuthenticationClient = WebAuthenticationClient()
```

Comment:
Did we want to consider making the class name similar to the Swift [WebAuthentication](https://okta.github.io/okta-mobile-swift/development/webauthenticationui/documentation/webauthenticationui/webauthentication)?

Alternatively, we might want to discuss how these UI environment abstraction classes will function in the future, particularly since we're going to be adding multiple for the Javascript client SDK (e.g. Vue, React, React Native, Electron, etc)

Reply:
Sure, I can rename it to WebAuthentication for consistency.